cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -318,7 +318,7 @@ def test_get_course_list_with_invalid_course_location(self, store):
         )
 
     @ddt.data(
-        (ModuleStoreEnum.Type.split, 3, 13),
+        (ModuleStoreEnum.Type.split, 4, 13),
```

Comment:
This was the only query count test to fail initially. Any huge concerns with that?